packages/react-native/ReactCommon/react/renderer/components/view/platform/macos/react/renderer/components/view/HostPlatformViewTraitsInitializer.h

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#pragma once
+
+#include <react/renderer/components/view/ViewProps.h>
+#include <react/renderer/core/ShadowNodeTraits.h>
+
+namespace facebook::react::HostPlatformViewTraitsInitializer {
+
+inline bool formsStackingContext(const ViewProps& props) {
+  return false;
+}
+
+inline bool formsView(const ViewProps& props) {
+  return props.focusable || props.enableFocusRing;
```

Comment:
Windows only sets focusable, not the latter. Should we do the same, or update windows?

Reply:
On the meta branch, these are also added to `formsStackingContext` instead... urge

Reply:
Looks like formsView includes all of formsStackingContext, so this can move. 

https://github.com/microsoft/react-native-macos/blob/18a6c7463db3b75613a24ba011fd80210cbea363/packages/react-native/ReactCommon/react/renderer/components/view/ViewShadowNode.cpp#L67

Reply:
enableFocusRing doesn't do anything if the view isn't focusable, so you probably dont need both, right?

Reply:
I don't think they bubble, so I'll add only `focusable` to `formsView` to match windows and android.